src/System.Management.Automation/engine/InternalCommands.cs

```diff
@@ -295,12 +295,12 @@ protected override void ProcessRecord()
                                 else
                                 {
                                     // we write null out because:
-                                    // PS C:\> �$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}�
+                                    // PS C:\> �$null | ForEach-object {$_.aa} | ForEach-Object {$_ + 3}�
```

Comment:
Ah, @SteveL-MSFT  please review the whole file - unicode chars?

Reply:
Will fix and replace the 'fancy quote' with a 'regular quote' character